src/library_browser.js

```diff
@@ -747,22 +747,22 @@ var LibraryBrowser = {
       }
       if (((document['fullscreenElement'] || document['mozFullScreenElement'] ||
            document['msFullscreenElement'] || document['webkitFullscreenElement'] ||
-           document['webkitCurrentFullScreenElement']) === canvas.parentNode) && (typeof screen != 'undefined')) {
+           document['webkitCurrentFullScreenElement']) === canvas.parentNode) && (typeof screen !== 'undefined')) {
```

Comment:
Since `typeof` is guaranteed to return a string, `==` is enough, and `===` just adds an extra byte. Maybe we should standardize on the shorter form, what do you think?

Reply:
Sure I guess we could go either way.. although `!==` and `===` seem to be the preferred style in JS world .. perhaps we can rely on closure to remove the extra bytes?

Reply:
Good question about closure here. I checked and it doesn't do optimizations like these as best I can tell.

Fair point that `===` is slightly better in general, as it's more precise. But where we can save a few bytes, it might be worth it to use `==` I think, and this is such a case since we do know the full type on both sides. Seems like closure can know it as well, though, so perhaps I'm missing something?

Reply:
OK, so I should change this PR to instead to "!=" and "==" throughout ?

Reply:
I think that makes the most sense, yeah.